src/pages/[spaceId]/index.tsx

```diff
@@ -31,7 +32,21 @@ export const getStaticPaths = async () => {
   }
 }
 
-const getLastPosts = async (commentIdsByPostId: string[][]) => {
+function getSpaceIdFromParam(paramSpaceId: string) {
+  const spaceIdOrTopic = paramSpaceId ?? getMainSpaceId()
+  let spaceId = spaceIdOrTopic
+  if (isNaN(parseInt(spaceIdOrTopic))) {
```

Comment:
isNaN(parseInt(spaceIdOrTopic)) looks like separate util function

Reply:
add isValidNumber util in strings util function